cmd/link/ocmrole/cmd.go

```diff
@@ -38,7 +38,7 @@ var Cmd = &cobra.Command{
 	Short:   "Link OCM role to specific OCM organization.",
 	Long:    "Link OCM role to specific OCM organization before you create your cluster.",
 	Example: ` # Link OCM role
-  rosa link ocm-role --role-arn arn:aws:iam::123456789012:role/ManagedOpenshift-OCM-Role`,
+  rosa link ocm-role --role-arn arn:aws:iam::123456789012:role/ManagedOpenshift-OCM-Role -y`,
```

Comment:
Any reason to change the example?

Reply:
Just to let users know there is a -y option